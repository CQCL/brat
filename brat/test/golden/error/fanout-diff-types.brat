f(Vec(Qubit, 2)) -o Qubit, Bit
f = { [/\] }
