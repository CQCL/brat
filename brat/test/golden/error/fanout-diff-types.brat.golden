Error in test/golden/error/fanout-diff-types.brat@2:5-13
f = { [/\] }
    ^^^^^^^^

  Type mismatch when checking [/\]
Expected: (b1 :: Bit)
But got:  (head :: Qubit)


